config/database.yml

```diff
@@ -0,0 +1,88 @@
+# PostgreSQL. Versions 9.3 and up are supported.
+#
+# Install the pg driver:
+#   gem install pg
+# On macOS with Homebrew:
+#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
+# On macOS with MacPorts:
+#   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
+# On Windows:
+#   gem install pg
+#       Choose the win32 build.
+#       Install PostgreSQL and put its /bin directory on your path.
+#
+# Configure Using Gemfile
+# gem "pg"
+#
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  username: postgres
+  password: asdf@123
+  # For details on connection pooling, see Rails configuration guide
```

Comment:
@team, I added the codes for my own. you can add yours. if you have some idea, to put this inside .gitignore. please share. so don't have to add everytime, some one push thier commits on github